apps/examples/src/examples/speech-bubble/SpeechBubble/SpeechBubbleHandle.tsx

```diff
@@ -0,0 +1,103 @@
+import { HandleControl, StateNode, Vec, track, useEditor } from '@tldraw/tldraw'
+import { useEffect } from 'react'
+import { SpeechBubbleShape } from './SpeechBubbleUtil'
+
+export const SpeechBubbleHandle = track(function SpeechBubbleHandle() {
+	const editor = useEditor()
+
+	useEffect(() => {
+		editor.root.find('select')!.addChild(DraggingSpeechBubble)
+	}, [editor])
+
+	if (!editor.isInAny('select.idle')) {
+		return null
+	}
+
+	const shape = editor.getOnlySelectedShape()
+	if (!shape || !editor.isShapeOfType<SpeechBubbleShape>(shape, 'speech-bubble')) {
+		return null
+	}
+
+	const transform = editor.getShapePageTransform(shape)
+	const localPosition = new Vec(
+		shape.props.tail.x * shape.props.w,
+		shape.props.tail.y * shape.props.h
+	)
+
+	return (
+		<HandleControl
+			position={transform.applyToPoint(localPosition)}
+			onPointerDown={(e) => {
+				e.preventDefault()
+				e.stopPropagation()
+
+				editor.root.transition('select.draggingSpeechBubble')
+			}}
+		/>
+	)
+})
+
+// missing from here:
+// - tool masks (what even is this?)
```

Comment:
Which tool should be shown as selected when this state is active? For example, creating an arrow technically goes to the select.dragging_handle, but we display the arrow tool as active.